README.md

```diff
@@ -60,6 +60,22 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
+## Configuration
+
+There are several options available through configuration.
```

Comment:
Thinking about this some more I think the defaults should be mentioned first and not as a side note as the end.  How about something like this:

------

By default `host` and `scheme` are pulled from the provided `conn` but can be overridden via configuration:

```elixir
config :jsonapi,
  host: "www.someotherhost.com",
  scheme: "https",
  underscore_to_dash: true
```

JSONAPI now recommends the use of dashes (`-`) in place of underscore (`_`) as a word separator, supporting this change is easy with the `underscore_to_dash` option which handles the conversion for you.

------

I think it _might_ be worth mentioning exactly _why_ we have an option called `underscore_to_dash` since that won't be immediately clear to others.